tuftool/Cargo.toml

```diff
@@ -34,6 +34,7 @@ tempfile = "3.1.0"
 url = "2.1.0"
 walkdir = "2.2.9"
 tough = { version = "0.5.0", path = "../tough", features = ["http"] }
+tokio = { version = "0.2.13", features = ["macros"] }
```

Comment:
You can probably drop `macros` here and not specify any features, since you're not using `#[tokio::main]`.